src/utilsUI.cpp

```diff
@@ -359,4 +359,37 @@ void resizeInFontHeight(QWidget *w, int width, int height)
 	w->resize(newSize);
 }
 
+/*!
+ * \brief Given font metrics return pixel size of a character
+ * \param[in] fm Font metrics
+ * \param[in] ch Character
+ * \returns Returns pixel size of character
+ */
+int getFmWidth(const QFontMetrics &fm, QChar ch)
+{
+#if QT_VERSION >= QT_VERSION_CHECK(5, 11, 0)
+	return fm.horizontalAdvance(ch);
+#else
+	return fm.width(ch);
+#endif
+}
+
+/*!
+ * \brief Given font metrics return pixel size of a text string
+ * \param[in] fm Font metrics
+ * \param[in] text Text string
+ * \param[in] len Only calculate width of the first len characters of the string. If not specified,
+ * then -1 is assumed which means calculate width of the whole string.
+ * \returns Returns pixel size of the text string
+ */
+int getFmWidth(const QFontMetrics &fm, const QString &text, int len)

```

Comment:
len is unused ?

Reply:
It is my omission. `len` should be an argument to `fm.horizontalAdvance`/`fm.width`. I fixed that and force-pushed the new version.

Ironically I was fixing warnings and did not notice my own bug that introduced a new warning. I guess I shouldn't write code while I am falling asleep.